addons/web/views/webclient_templates.xml

```diff
@@ -281,6 +281,9 @@
                 <t t-call-assets="web.assets_common" t-css="false"/>
                 <t t-call-assets="web.assets_backend" t-css="false"/>
                 <t t-call-assets="web.assets_backend_prod_only" t-css="false"/>
+                <script type="text/javascript">
+                    odoo.loader.checkAndReportErrors();
+                </script>
```

Comment:
Something like that: 
`
define(name, deps, factory) {
            if (typeof name !== "string") {
                throw new Error(`Invalid name definition: ${name} (should be a string)"`);
            }
            if (!(deps instanceof Array)) {
                throw new Error(`Dependencies should be defined by an array: ${deps}`);
            }
            if (typeof factory !== "function") {
                throw new Error(`Factory should be defined by a function ${factory}`);
            }
            if (!this.factories.has(name)) {
                this.factories.set(name, { deps, fn: factory });
                this.addJob(name);
            }
        }
            Promise.resolve().then(() => this.checkAndReportErrors());
`

Reply:
Yes, though you should probably only schedule this check if one is not already scheduled, so that you don't check for errors once per define in the same assets bundle